Add playtest-20221222 announcement page (#32)
OpenRA/OpenRAWebsiteV3: news/_posts/2022-12-23-playtest-20221223.md

---
kind: article
title: "Playtest 20221223"
author: "PunkPun"

> created_at: 2022-12-22 23:00 +0000

Reviewer: This will also want to be updated I guess.

disqus_id: "playtest-20221223"
permalink: "/news/playtest-20221223/"
---

<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
<div style="margin: -10px 5px" markdown="1">

Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):

> * Fixed a few pathfinder crashes

Reviewer: ```suggestion
* Fixed several pathfinder-related crashes
```
? (To not have "Fixed a few" repeated twice in a row, unless that is intentional.)

> * Fixed a few Red Alert missions

Reviewer: ```suggestion
* Fixed the winning condition in a few Red Alert missions
```

> * Fixed carryalls in rare cases destrying their cargo

Reviewer: ```suggestion
* Fixed carryalls in rare cases destroying their cargo
```

* Added caryalls to the D2K spectator economy statistics
* Added new maps to the D2K map pool

> * A few other minor fixes

Reviewer: This feels a bit off. Maybe we can reword it to something along the lines of `Further fixes to the map editor and core engine`?

Author: Changed it to `Other minor fixes`

> We expect this will be our final playtest before a full release on New Year's, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or on [GitHub](https://github.com/OpenRA/OpenRA/issues).

Reviewer: New Year's *Day*? So first of January. Or New Year's Eve? I wouldn't mention this at all but just write
```suggestion
We expect this will be our final playtest before a full release, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or on [GitHub](https://github.com/OpenRA/OpenRA/issues).
```

---
kind: article
title: "Playtest 20221223"
author: "PunkPun"
created_at: 2022-12-22 23:00 +0000
disqus_id: "playtest-20221223"
permalink: "/news/playtest-20221223/"
---

<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
<div style="margin: -10px 5px" markdown="1">

Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):

* Dropped support for macOS 10.9 and 10.10

> * Updated macOS packaging, linux and macOS dependencies

Reviewer 2: * Added macOS universal binaries for Apple Silicon

and put this before the dropped support line.

Author: ok

---
kind: article
title: "Playtest 20221223"
author: "PunkPun"
created_at: 2022-12-23 18:00 +0000

> disqus_id: "playtest-20221223"

Reviewer 2: We would generally keep the previous comments for posts that are tacking extra info on top of a previous one.

Author: makes sense

permalink: "/news/playtest-20221223/"
---

<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
<div style="margin: -10px 5px" markdown="1">

> Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):

Reviewer: ```suggestion
Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog/e5fda938ada5be95883b9631e2dd1a3609fead45):
```

* Added macOS universal binaries for Apple Silicon
* Dropped support for macOS 10.9 and 10.10
* Updated macOS packaging, linux and macOS dependencies
* Fixed several pathfinder-related crashes
* Fixed the victory condition in a few of the Red Alert missions
* Fixed carryalls in rare cases destroying their cargo

> * Added caryalls to the D2K spectator economy statistics

Reviewer: ```suggestion
* Added caryalls to the Dune 2000 spectator economy statistics
```

Author: Feels weird but formal style I guess

> * Added new maps to the D2K map pool

Reviewer: ```suggestion
* Added new maps to the Dune 2000 map pool
```
